plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -52,6 +52,42 @@ Example input:
 
 ### Actions
 
+#### Get File Quarantine Status
+
+This action is used to get File Quarantine Status.
```

Comment:
This description isn't correct - it should fix itself after you regenerate the plugin (based on the current plugin.spec.yaml file).

Reply:
I regenerated the plugin and get the same value here. What should it be?